scripts/exregional_bias_correction_pm25.sh

```diff
@@ -111,9 +120,9 @@ if [ -d "${DATA}/data/bcdata.${yyyymm}" ]; then
   cp -rL "${AQM_AIRNOW_HIST_DIR}/bcdata.${yyyymm}/airnow" "${DATA}/data/bcdata.${yyyymm}"
   cp -rL "${AQM_AIRNOW_HIST_DIR}/bcdata.${yyyymm}/interpolated" "${DATA}/data/bcdata.${yyyymm}"
 fi
-
+cd ${DATA}
 # Retrieve real-time airnow data for the last three days
-if [ "${DO_REAL_TIME}" = "TRUE" ]; then
+#if [ "${DO_REAL_TIME}" = "TRUE" ]; then
```

Comment:
Remove this line if not necessary.

Reply:
Will have a global developer code cleanup as there are still many other developer code in the AQM package at this time.